team/team.json

```diff
@@ -0,0 +1,277 @@
+{
+	"forth": [
+		{
+			"profile": "Pranjali.png",
+			"name": "Pranjali Katiyar",
+			"designation": "Chief Executive Member",
+			"links": [
+				{
+					"website": "linkedin",
+					"url": "https://www.linkedin.com/in/pranjali-katiyar-1aa15a15a/"
+				},
+				{
+					"website": "github",
+					"url": "https://github.com/pr-jli"
+				}
+			]
+		},
+		{
+			"profile": "prakhar.png",
+			"name": "Prakhar Varshney",
+			"designation": "CTC Member, Head of Club: Technical",
```

Comment:
Remove `Member`, and Write HOC's as we have in the current version, 
![image](https://user-images.githubusercontent.com/42200276/99246123-c6901180-282a-11eb-9dc3-66615560ded6.png)

Reply:
Change this for all HOC's